packages/user-interface/src/components/CasesList.tsx

```diff
@@ -15,8 +15,7 @@ interface Props {
   error?: boolean;
   errorTranslationId?: string;
   emptyTranslationId?: string;
-  showTitle?: boolean;
-  titleTranslationId?: string;
+  titleTranslationId?: string | false;
```

Comment:
Het voelt een beetje apart om een boolean waarde te kunnen meegeven aan een id property. Zou dat niet beter string | undefined kunnen zijn?

Reply:
Nee want met undefined pakt hij de default: titleTranslationId = "casesList.title",

Reply:
null zou volgens mij wel moeten werken

Reply:
Dan is de vraag wil je null of false. Dan vind ik false mooier ;)